src/databricks/labs/ucx/framework/tasks.py

```diff
@@ -168,8 +208,8 @@ def trigger(*argv):
 
     log_readme = log_path.joinpath("README.md")
     if not log_readme.exists():
-        # this may race when run from multiple tasks, but let's accept the risk for now.
-        with log_readme.open(mode="w") as f:
+        # this may race when run from multiple tasks, therefore it must be multiprocess safe
+        with exclusive_open(str(log_readme), mode="w") as f:
```

Comment:
is there a method for this on `pathlib.Path`?

Reply:
do you mean to get the file as string?